Code/ElastixTransformixWrappers/include/sitkElastixImageFilter.h

```diff
@@ -55,121 +58,264 @@ class SITKElastix_EXPORT ElastixImageFilter
+
+  /** \brief Adds a parameter specified by \p key, with the specified value to the parameter map at the specified (zero-based) \p index. */
   SITK_RETURN_SELF_TYPE_HEADER AddParameter( const unsigned int index, const std::string key, const std::string value );
+
+  /** \brief Adds a parameter specified by \p key, with the specified values to all parameter maps. */
   SITK_RETURN_SELF_TYPE_HEADER AddParameter( const std::string key, const std::vector< std::string > value );
+
+  /** \brief Adds a parameter specified by \p key, with the specified values to the parameter map at the specified (zero-based) \p index. */
   SITK_RETURN_SELF_TYPE_HEADER AddParameter( const unsigned int index, const std::string key, const std::vector< std::string > value );
+
+  /** \brief Intended to retrieve the values of the parameter specified by \p key, when there is only one parameter map. */
   std::vector< std::string > GetParameter( const std::string key );
+
+  /** \brief Retrieves the values of the parameter specified by \p key, from the parameter map at the specified (zero-based) \p index. */
   std::vector< std::string > GetParameter( const unsigned int index, const std::string key );
+
+  /** \brief Removes the parameter specified by \p key from all parameter maps. */
   SITK_RETURN_SELF_TYPE_HEADER RemoveParameter( const std::string key );
+
+  /** \brief Removes the parameter specified by \p key from the parameter map at the specified (zero-based) \p index. */
   SITK_RETURN_SELF_TYPE_HEADER RemoveParameter( const unsigned int index, const std::string key );
 
+  /** \brief Specifies the initial transformation by the specified transform parameter file name. */
   SITK_RETURN_SELF_TYPE_HEADER SetInitialTransformParameterFileName( const std::string initialTransformParmaterFileName );
+
+  /** \brief Returns the initial transform parameter file name. */
   std::string GetInitialTransformParameterFileName();
+
+  /** \brief Clears the initial transform parameter file name. */
   SITK_RETURN_SELF_TYPE_HEADER RemoveInitialTransformParameterFileName();
 
+  /** \brief Reads the parameter file specified by \p filename, and returns its content as a parameter map. */
   std::map< std::string, std::vector< std::string > > ReadParameterFile( const std::string filename );
+
+  /** \brief Writes a parameter map to the file, specified by \p filename. */
   SITK_RETURN_SELF_TYPE_HEADER WriteParameterFile( const std::map< std::string, std::vector< std::string > > parameterMap, const std::string filename );
 
+  /** \brief Executes the registration, and returns the result image.
+   * \note Before executing, the number of moving images must equal the number of fixed images,
+   * the number of fixed masks must be either zero, or one, or equal to the number of fixed images,
+   * and the number of moving masks must be either zero, or one, or equal to the number of moving images
+   * /
```

Comment:
The '* /' with the space seems to be causing a compilation error.
../../Code/ElastixTransformixWrappers/include/sitkElastixImageFilter.h:303:3: warning: "/*" within comment [-Wcomment]
[18351](https://github.com/SimpleITK/SimpleITK/actions/runs/3128399687/jobs/5076187945#step:6:18352)
  303 |   /** \brief Returns all transform parameter maps. */
[18352](https://github.com/SimpleITK/SimpleITK/actions/runs/3128399687/jobs/5076187945#step:6:18353)
      |    
[18353](https://github.com/SimpleITK/SimpleITK/actions/runs/3128399687/jobs/5076187945#step:6:18354)
../../Code/ElastixTransformixWrappers/src/sitkElastixImageFilter.cxx:621:1: error: no declaration matches ‘itk::simple::Image itk::simple::ElastixImageFilter::Execute()’
[18354](https://github.com/SimpleITK/SimpleITK/actions/runs/3128399687/jobs/5076187945#step:6:18355)
  621 | ElastixImageFilter
[18355](https://github.com/SimpleITK/SimpleITK/actions/runs/3128399687/jobs/5076187945#step:6:18356)
      | ^~~~~~~~~~~~~~~~~~